.github/workflows/release-please.yaml

```diff
@@ -13,5 +13,37 @@ permissions:
 jobs:
   release-please:
     runs-on: ubuntu-latest
+    permissions:
+      packages: write
+      contents: read
     steps:
+      - uses: actions/checkout@v4
+
+      # Extract Rust crate version
+      - id: extract
+        run: |
+          version=$(grep '^version' crates/multipathd-structs/Cargo.toml | head -n1 | cut -d '"' -f2)
+          echo "version=$version" >> "$GITHUB_OUTPUT"
+      # Update Helm Chart.yaml appVersion
+      - run: |
+          sed -i "s/^appVersion:.*/appVersion: \"${{ steps.extract.outputs.version }}\"/" charts/prometheus-dellhw-exporter/Chart.yaml
+      # Commit the change
+      - run: |
+          git config user.name "github-actions"
+          git config user.email "github-actions@github.com"
+          git add charts/prometheus-dellhw-exporter/Chart.yaml
+          git commit -m "chore(chart): sync appVersion to ${{ steps.extract.outputs.version }}" || echo "No changes to commit"
+          git push || echo "No changes to push"
+
```

Comment:
So what you can do here is ask `release-please` to include an extra file that is keeps updated.

See the `extra-files` option :)

https://github.com/vexxhost/atmosphere.common/blob/main/release-please-config.json#L7-L10